docs/touch-up.sh

```diff
@@ -11,10 +11,10 @@ for i in ls -l /docs/content/*
         y=${i##*/}
         find $i -type f -name "*.md" -exec sed -i.old \
         -e '/^<!.*metadata]>/g' \
+	-e '/(https:\/\//!s/(\([^)]*\.md\))/({{< relref "\1" >}})/g' \
```

Comment:
If you add or change the individual builds notifying me. We should be using the same scripts every place.  The place to add the scripts changes is to the docker/docs.docker.com  --- docs-base is theme.